src/util/state.ts

```diff
@@ -702,9 +705,9 @@ return 0;
     return parseInt(done || "0");
   }
 
-  async numSeen() {
-    return await this.redis.scard(this.skey);
-  }
+  // async numSeen() {
+  //   return await this.redis.scard(this.skey);
+  // }
```

Comment:
I guess this isn't used anymore, but maybe we just remove it? Pretty straightforward to add back if needed.